docs/Chain_parameters.md

```diff
@@ -11,36 +11,40 @@ Note: If you run the validator (`arb-validator` or `arb_deploy.py`) without any
 
 The parameters are:
 
--   stake requirement: This specifies how many Eth a validator must deposit as a stake.
+-   stake requirement: This specifies how much currency a validator must deposit as a stake.
     Making this bigger strengthens deterrence against malicious behavior by validators,
     but it also increases the amount of capital that validators have to lock up.
 -   grace period: This is the time period allowed for one validator to challenge an assertion made by another validator.
     Making this bigger makes certain attacks against the system more difficult,
     but it also slows down confirmation of transactions.
--   max execution steps: This is the maximum number of AVM instructions that can be executed in a single assertion.
-    Making this bigger increases the efficiency of the protocol (with diminishing returns as it gets really big),
-    but it can also slow down confirmation of transactions.
--   speed limit: This is the maximum amount of ArbGas consumption that can occur per second of real time.
-    Making this bigger lets the contracts in your ArbChain run faster,
-    but making it too big risks overwhelming validators who can't keep up.
-
-## Recommended parameters for production testing
+-   speed limit: This controls how fast computations on the chain can go, to make sure that every validator can keep up with 
+    computations. This is essentially the speed of the slowest validator you expect to have.  The speed is scaled so that 1.0
+    is the speed of a typical developer laptop. For example, if you are using half of a laptop's capacity to run the
+    validator, you should set the speed limit to 0.5.
+-   max assertion size: This is the largest amount of computation that you want to include in one assertion. If the chain is
+    running at 100% capacity, the updated state of the chain will be posted to the Ethereum chain this often.
+-   max time bounds width: This is how much time an assertion has to be accepted on-chain, before it is discarded. It is also
+    the maximum clock lag that can exist between the chain's clock (as returned by `block.number` in Solidity) and the block
+    number on the main Ethereum chain.
+
+## Recommended parameters 
 
 For executing in a production-like setting, we suggest the following parameters:
 
--   stake requirement: 2% of total value at stake in your ArbChain
```

Comment:
is this more accurate? maybe we should keep this and just say we default to 1 in the deployment tool?

Reply:
I'll add this back in as commentary, that this is what we expect to recommend for production, eventually.